### ISSUE TITLE
Factory and stack features are not supported in Workspace.Next

### ISSUE THREAD

**Issue by garagatyi** [eclipse-che/che#10368]
### Description
Right now Workspace.Next using workspace attributes to store a list of Che features. But we don't have attributes in the Factory object, so can't use Factory with Workspace.Next for the time being.
We need to find a way to add Workspace.Next capabilities to factories. Would be awesome to have it in walking skeleton https://github.com/eclipse/che/issues/10123.

### Reproduction Steps

**OS and version:**    

**Diagnostics:**

**Comment by garagatyi** [comment]
@slemeur since you are a known Factory and Workspace.Next fan here is something you might be interested in!

**Comment by garagatyi** [comment]
After the discussion with Florent, Anna, Sergii, Serhii, and Yevhen we decided to add an `attributes` field to the workspace config model object. This to make our workspace model object contains two fields with the same name - one on the whole workspace level and another one on the workspace config level.
We also decided that attributes from config won't be added to the whole workspace attributes to not bring more confusion.
This solution would allow us to have walking skeleton stuff in both Factory and Stack.
We also mentioned that solution might be temporary since, eventually, we will add che editor and che plugins fields in the workspace model directly. So, it might make sense to mark this field with Beta annotation or so.
